app/controllers/admin/questions_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Admin::QuestionsController < ApplicationController
+  def index
+    @bank_questions = Question.for_banks
+    @partner_questions = Question.for_partners
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.create(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to create question."
```

Comment:
Let's include information on why it didn't succeed. Perhaps we can use the errors from `@question`?

Reply:
@edwinthinks The only field where a validation error message isn't already displayed is the rich text answer field. Would something like the following work?

 
![Screenshot (369)](https://user-images.githubusercontent.com/42154066/149606336-63e0593a-90b3-4d20-98e9-d9176de66bed.png)

![Screenshot (370)](https://user-images.githubusercontent.com/42154066/149606344-6b2b019b-dacf-4ada-8191-8c5888a233ec.png)

![Screenshot (371)](https://user-images.githubusercontent.com/42154066/149606348-1f6dc413-5fe9-49a2-8f70-1066fe7834be.png)

![Screenshot (372)](https://user-images.githubusercontent.com/42154066/149606359-9f6b61b4-dadd-4d13-9c63-773005d89d7b.png)